hnn_core/tests/test_dipole.py

```diff
@@ -56,7 +56,27 @@ def test_dipole(tmpdir, run_hnn_core_fixture):
                        "average of 2 trials"):
         dipole_avg = average_dipoles([dipole_avg, dipole_read])
 
+    # average an n_of_1 dipole list
+    single_dpl_avg = average_dipoles([dipole])
+    for dpl_key in single_dpl_avg.data.keys():
+        assert_allclose(
+            dipole_read.data[dpl_key],
+            single_dpl_avg.data[dpl_key],
+            rtol=0,
+            atol=0.000051)
+
+    # average dipole list with one dipole object and a zero dipole object
+    n_times = len(dipole_read.data['agg'])
+    dpl_null = Dipole(np.zeros(n_times, ), np.zeros((n_times, 3)))
+    dpl_1 = [dipole, dpl_null]
+    dpl_avg = average_dipoles(dpl_1)
+    for dpl_key in dpl_avg.data.keys():
+        assert_allclose(dpl_1[0].data[dpl_key] / 2., dpl_avg.data[dpl_key])
+
+    # test postproc
+
```

Comment:
```suggestion
```

seems like a spurious comment not related to PR

Reply:
This comment was in the code before I had started my PR. While rebasing, I went back to an old commit to copy and paste the changes that I had made including the comment. I assumed that the comment was made by a previous commit and referenced the subsequent test cases. Should I keep this comment or just remove it?

Reply:
You can go ahead and remove it no longer exists on the master branch. Before #372 was merged it was here: https://github.com/jonescompneurolab/hnn-core/blob/5ade97af0d6da7ab223442ca7cd954b774156d3f/hnn_core/tests/test_dipole.py#L51
 but it's been replaced now